azurelinuxagent/common/osutil/default.py

```diff
@@ -267,11 +276,22 @@ def enable_firewall(self, dst_ip, uid):
 
         try:
             wait = self.get_firewall_will_wait()
-
             # If the DROP rule exists, make no changes
             try:
                 drop_rule = _get_firewall_drop_command(wait, "-C", dst_ip)
                 shellutil.run_command(drop_rule)
+                try:
+                    accept_rule = _get_firewall_accept_command_nonroot_tcp(wait, "-C", dst_ip)
```

Comment:
On the same note, can you also create a new parameter for `-C` parameter and add a small comment on what it does (checks if the rule exists in the chain).

Reply:
This logic moved under Update.py and addressed your comment there.